spidermon/contrib/actions/telegram/__init__.py

```diff
@@ -0,0 +1,119 @@
+from __future__ import absolute_import
+
+import ast
+import json
+import logging
+import requests
+
+import six
+
+from spidermon.contrib.actions.templates import ActionWithTemplates
+from spidermon.exceptions import NotConfigured
+
+logger = logging.getLogger(__name__)
+
+
+class SimplyTelegramClient:
+    send_message_api = "https://api.telegram.org/bot{token}/sendMessage?chat_id={chat_id}&parse_mode=Markdown&text={text}"
```

Comment:
According to Telegram APIs doc, only chat_id and text are required: 
https://core.telegram.org/bots/api#sendmessage

Reply:
Yes, but I am using `parse_mode=Markdown` to format the message using Jinja.